Test APIs.ipynb

```diff
@@ -255,7 +272,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.5"
+   "version": "3.7.4"
```

Comment:
I'm not sure whether this should be updated or not. I guess not, but let me know, I'll revert it.